docs/_assets/snippets.js

```diff
@@ -27,15 +28,23 @@ function MaterialComponentsSnippets() {
  */
 MaterialComponentsSnippets.prototype.init = function() {
```

Comment:
will this ever be inited outside of window.onload? 
If so should we think about providing a destroy method to clean these listeners?